service/controller/v13/encrypter/vault/vault.go

```diff
@@ -99,6 +96,28 @@ func (e *Encrypter) CreateKey(ctx context.Context, customObject v1alpha1.AWSConf
 		return microerror.Mask(invalidHTTPStatusCodeError)
 	}
 
+	// we need to make the key erasable.
```

Comment:
Nitpicking but comment should start with a capital. Also could be deletable rather than erasable.

Reply:
Sure, done.